nixos/modules/services/matrix/lk-jwt-service.nix

```diff
@@ -0,0 +1,93 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.lk-jwt-service;
+in
+{
+  meta.maintainers = [ lib.maintainers.quadradical ];
+  options.services.lk-jwt-service = {
+    enable = lib.mkEnableOption "Enable lk-jwt-service";
+    package = lib.mkPackageOption pkgs "lk-jwt-service" { };
+
+    livekitUrl = lib.mkOption {
+      type = lib.types.strMatching "^wss?://.*";
+      example = "wss://example.com/livekit/sfu";
```

Comment:
I think this should default to `"wss://127.0.0.1:${config.services.livekit.settings.port}"`

you could also only let it default to it if `config.services.livekit.enable` is true and add an assert that either config.services.livekit.enable has to be true or livekitUrl must be set